hadoop-tools/hadoop-azure-datalake/src/main/java/org/apache/hadoop/fs/adl/AdlFileSystem.java

```diff
@@ -65,6 +66,8 @@
 import org.apache.hadoop.util.Progressable;
 import org.apache.hadoop.util.ReflectionUtils;
 import org.apache.hadoop.util.VersionInfo;
+import org.slf4j.Logger;
```

Comment:
should be ordered according to the hadoop import ordering. I know we get this wrong, and once its wrong we don't like to fix (it breaks all patches and cherry picking), but it's good to try